src/rhsm/profile.py

```diff
@@ -149,7 +149,7 @@ def __generate(self):
             all_module_list = modules.getModulePackages()
 
             for module_pkg in all_module_list:
-                status = "unknown"
+                status = "default"
                 active = False
```

Comment:
The BZ may not be clear about this, but I believe the active flag is incorrect as well. A module may be active even if it hasn't been explicitly enabled. On line 157, we're only setting the active flag if the module is also enabled, which means we'll treat some modules as inactive even though they're active and in the default state.

Reply:
Hi,
Thanks for feedback. It is written in the BZ. I only overlooked that. It should be fixed ATM.

Reply:
Looks good to me. Ty!